speaker/main.go

```diff
@@ -429,3 +364,9 @@ type Protocol interface {
 	DeleteBalancer(gokitlog.Logger, string, string) error
 	SetNode(gokitlog.Logger, *v1.Node) error
 }
+
+// Speakerlist represents a list of healthy speakers.
+type Speakerlist interface {
```

Comment:
Is there a reason we use `Speakerlist` here but `SpeakerList` elsewhere?

Reply:
No, switched them all to SpeakerList